<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <widget class="QLabel" name="label_calc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_credit">
    <item>
     <widget class="QLabel" name="label_credit_data">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Данные о кредите</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_answer">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Результаты расчета</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>110</y>
     <width>20</width>
     <height>491</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_pay">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>95</number>
   </property>
   <property name="format">
    <string>%p% credit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_credit">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>75</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_pay">
      <property name="text">
       <string>Ежемесячный платеж</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_pay_enter">
      <property name="styleSheet">
       <string notr="true">QLabel {
border:1px solid gray;
background-color:#FFFFFF;
}

</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_overpay">
      <property name="text">
       <string>Переплата по кредиту</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_overpay_enter">
      <property name="styleSheet">
       <string notr="true">QLabel {
border:1px solid gray;
background-color:#FFFFFF;
}

</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_total_pay">
      <property name="text">
       <string>Общая выплата</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_total_pay_enter">
      <property name="styleSheet">
       <string notr="true">QLabel {
border:1px solid gray;
background-color:#FFFFFF;
}

</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>371</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6_credit">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Credit</string>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Overpayment</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>487</y>
     <width>89</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип платежа</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>510</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radio_type_annuitet">
      <property name="text">
       <string>Аннуитетный</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_type_diff">
      <property name="text">
       <string>Дифференцированный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>30</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_5_credit">
      <item row="0" column="0">
       <widget class="QLabel" name="label_sum">
        <property name="text">
         <string>Сумма кредита</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="line_sum">
        <property name="placeholderText">
         <string>1000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2_credit" rowminimumheight="1,2">
      <item row="0" column="1">
       <widget class="QLabel" name="label_currency">
        <property name="text">
         <string>Тип валюты</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_curr_rub">
        <property name="text">
         <string>Рубль</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radio_curr_dollar">
        <property name="text">
         <string>Доллар</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radio_curr_euro">
        <property name="text">
         <string>Евро</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_3_credit">
      <item row="0" column="0">
       <widget class="QLabel" name="label_time">
        <property name="text">
         <string>Срок, в месяцах</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_rate">
        <property name="text">
         <string>Ставка, в %</string>
        </property>
        <property name="margin">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="line_time">
        <property name="placeholderText">
         <string>12</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_rate">
        <property name="placeholderText">
         <string>9.5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
